api/managementpb/backup/backups.proto

```diff
@@ -58,12 +62,130 @@ message RestoreBackupResponse {
   string restore_id = 1;
 }
 
+// ScheduledBackup represents scheduled task for backup.
+message ScheduledBackup {
+  // Machine-readable ID.
+  string scheduled_backup_id = 1;
+  // Machine-readable service ID.
+  string service_id = 2;
+  // Service name.
+  string service_name = 3;
+  // Machine-readable location ID.
+  string location_id = 4;
+  // Location name.
+  string location_name = 5;
+  // How often backup will be run in cron format.
+  string cron_expression = 6;
+  // First backup wouldn't happen before this time.
+  google.protobuf.Timestamp start_time = 7;
+  // Artifact name.
+  string name = 8;
```

Comment:
probably we should have a pattern here, so every new artifact will have a different name.

Reply:
for now, i want just add timestamp after this name. Still we didn't had discussion how to handle this "autogenerate name"